util/src/main/scala/com/scalableminds/util/geometry/BoundingBox.scala

```diff
@@ -77,13 +97,23 @@ object BoundingBox {
     else
       None
 
-  def combine(bbs: List[BoundingBox]): BoundingBox =
+  def union(bbs: List[BoundingBox]): BoundingBox =
     bbs match {
       case head :: tail =>
-        tail.foldLeft(head)(_ combineWith _)
+        tail.foldLeft(head)(_ union _)
       case _ =>
         BoundingBox(Vec3Int(0, 0, 0), 0, 0, 0)
```

Comment:
Maybe this should be None as well then? But it probably just makes other checks necessary that can be avoided this way.

Reply:
You’re right, that would be cleaner. However, an empty union bbox only happens when an empty list of boxes is passed, which is way more rare than for intersection, where this happens on different bboxes being passed that just happen not to intersect. I think I won’t touch the union code in the scope of this PR. Maybe it will come up again, then I’m open to changing this to None as well :)